lib/kit.js

```diff
@@ -28,6 +25,22 @@ class CoinsenceKit {
     return new CoinsenceKit(provider, signer, options);
   }
 
+  async init() {
+    return this.provider.getNetwork().then((network) => {
+      this.address = KitAddress[network.chainId.toString()];
```

Comment:
Id' only do that if address is not set. otherwise it can not be manually configured. 
```suggestion
      this.address = this.address || KitAddress[network.chainId.toString()];
```